### ISSUE TITLE
利用ncnn2mem 转成二进制的之后加载（net.load_param_bin）就会报：layer -1 not exists or registered

### ISSUE THREAD

**Issue by ChisenZhang** [xiangweizeng/darknet2ncnn#14]
darknet 模型，转成ncnn模型后直接调用没问题；利用ncnn2mem 转成二进制的之后加载（net.load_param_bin）就会报：layer -1 not exists or registered 用net.load_param加载就会报param is too old, please regenerate的错误？ 什么原因？

**Comment by ChisenZhang** [comment]
模型长这样

[net]
#Training
#batch=16
#subdivisions=4

Testing
batch=1
subdivisions=1

height=256
width=256
channels=3

min_crop=128
max_crop=256
burn_in=1000
learning_rate=0.0005 # 初始学习率
policy=poly
power=4
max_batches=800000 # 最大bath
momentum=0.9 # 动量
decay=0.0005 # 权重衰减正则项

angle=7 # 旋转角度
hue=.1 # 色调
saturation=.75 # 饱和度
exposure=.75 # 曝光量
#aspect=.75 #

adam=1
B1=0.9
B2=0.999
eps=1e-7

[convolutional]
filters=16
size=7
stride=4
pad=1
batch_normalize=1
activation=leaky

#S1
[convolutional]
filters=32
size=1
stride=1
pad=1
batch_normalize=1
activation=leaky

#DWConv
[convolutional]
filters=32
size=3
stride=2
groups=32
pad=1
batch_normalize=1
activation=leaky

[convolutional]
filters=16
size=1
stride=1
pad=1
batch_normalize=1
activation=linear

#S2
[convolutional]
filters=32
size=3
stride=2
pad=1
batch_normalize=1
activation=leaky

#DWConv
[convolutional]
filters=64
size=1
stride=1
pad=1
batch_normalize=1
activation=leaky

[avgpool]

[convolutional]
filters=2
size=1
stride=1
pad=1
activation=linear

[softmax]
groups=1

**Comment by xiangweizeng** [comment]
官方的工具目前不支持自定义网络层， 需调整后方可支持，但目前本工具还未支持，后续考虑加入相关的功能支持。
